pyro/distributions/testing/special.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2014, Salesforce.com, Inc.  All rights reserved.
```

Comment:
This file is ported from https://github.com/posterior/goftests/blob/master/goftests/utils.py
Differences include:
- port to Python 3